src/config/config.rs

```diff
@@ -131,7 +135,7 @@ impl Type {
 pub struct GraphQL {
   pub schema: RootSchema,
   pub types: BTreeMap<String, Type>,
-  pub unions: Option<Vec<Union>>,
+  pub unions: BTreeMap<String, Union>,
```

Comment:
I think if we wish to store the name of the Union type as a Key in BTreeMap, we should remove the field `name` from `Union`

Reply:
BTW I am wondering if the change in union should be in the scope of this PR. However this is a good improvement so happy to merge :)